lib/environment.ts

```diff
@@ -0,0 +1,5 @@
+export function clientInfo(): string {
+  let version = require('../package.json').version
+  let name = require('../package.json').name
```

Comment:
When reviewing this, I realized these lines can be `const` and it's a string coming back. Could you make a change like this one https://github.com/percy/percy-cypress/commit/7253b2d073f730ab83760875203a1c4aee26498d here too?

Reply:
Sure. Fixed here, will send PR for cypress.

Reply:
(Misread that. You made the Cypress change, I'll mirror that here.)